src/main/features/claim/helpers/featuresBuilder.ts

```diff
@@ -9,7 +9,7 @@ const featureTogglesClient: FeatureTogglesClient = new FeatureTogglesClient()
 export class FeaturesBuilder {
   static readonly MEDIATION_PILOT_AMOUNT = 500
   static readonly LA_PILOT_THRESHOLD = 300
-  static readonly JUDGE_PILOT_THRESHOLD = 1000
+  static readonly JUDGE_PILOT_THRESHOLD = 10000
```

Comment:
is there any test to figure out if it consider case as JDDO if the claim amount is within 300-10000?

Reply:
There are 2 tests performed in src/test/features/claim/helpers/featuresBuilder.ts:

1. it(`should add judge pilot eligible to features if amount <= ${FeaturesBuilder.JUDGE_PILOT_THRESHOLD} and flag is set`, async () => {
2. it(`should not add judge pilot eligible to features if amount > ${FeaturesBuilder.JUDGE_PILOT_THRESHOLD}`, async () => {

Reply:
thanks @justin-r-oliver